<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>summonedWindow</class>
 <widget class="QMainWindow" name="summonedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Summoned Kalk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nameLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Summoned</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="HPLine1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="HPLine2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="HPLabel2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QLabel" name="baLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BaseAttack</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLine1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLine2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fightButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>FIGHT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="baLine1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="displayRes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Select or create an unit</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="baLabel2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BaseAttack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel2">
    <property name="geometry">
     <rect>
      <x>683</x>
      <y>210</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Summoned</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeKalkHero">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ChangeKalk</string>
    </property>
   </widget>
   <widget class="QLabel" name="HPLabel1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>HP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="baLine2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="summList">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>400</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>UNIT POOL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select1Button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select unit 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select2Button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Unit 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dpsButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dpsButton_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dbtButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DamageByTime</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dbtButton2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DamageByTime</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>summList</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>summonedWindow</receiver>
   <slot>onSUMItemClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSUMItemClicked(QListWidgetItem*)</slot>
 </slots>
</ui>
